restconnector/RequestHandler.cpp

```diff
@@ -25,9 +27,13 @@ RequestHandler::RequestHandler( const std::string& publisherSchema,
     , publisher_( new ::zeq::Publisher( lunchbox::URI( publisherSchema ) ) )
     , listening_( true )
     , listeningThread_( new boost::thread( boost::bind( &RequestHandler::listen_, this ) ) )
+    , renderingRessourceStrarted_( false )
 {
-    subscriber_->registerHandler( ::zeq::hbp::EVENT_IMAGEJPEG,
-                                  boost::bind( &RequestHandler::onImageJPEGEvent_, this, _1 ));
+    ZeqEventDescriptor eventDescriptor;
+    eventDescriptor.zeqEvent_ = ::zeq::hbp::EVENT_IMAGEJPEG;
+    eventDescriptor.schema_ = ::zeq::hbp::SCHEMA_IMAGEJPEG;
```

Comment:
We have weirds errors if we create a constructor when we try to insert this struct into a std::map...  
no matching function for call to 'restconnector::RequestHandler::ZeqEventDescriptor::ZeqEventDescriptor()

If we create a default constructor, a empty struct is inserted...
Do you know a work around ?